servers/physics_3d/area_3d_sw.cpp

```diff
@@ -163,6 +163,18 @@ void Area3DSW::set_param(PhysicsServer3D::AreaParameter p_param, const Variant &
 		case PhysicsServer3D::AREA_PARAM_PRIORITY:
 			priority = p_value;
 			break;
+		case PhysicsServer3D::AREA_PARAM_WIND_FORCE_MAGNITUDE:
+			wind_force_magnitude = p_value;
```

Comment:
Same here, there should be an error for negative values for cases where the physics server is called directly.